docs/how-to-guides/auto-update-flatcar.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Auto-updating Flatcar Container Linux
+weight: 10
+---
+
+## Introduction
+
+Lokomotive supports Flatcar Container Linux only. Flatcar Linux Update Operator (FLUO) can keep the operating system updated every time there is a new Flatcar release. FLUO is a supported component of the Lokomotive's component stack. This document will show how to install it and disable its auto-update feature for specific nodes.
+
+## Prerequisites
+
+- A Lokomotive cluster accessible via `kubectl`.
+
+## Steps
+
+### Step 1: Disable auto-update on nodes
+
+If you want to update the Flatcar OS of particular nodes in a controlled fashion, there is a way to disable such updates. Disabling updates can come handy when the workloads run by these machines are storage related or ingress nodes, where you cannot afford the slightest loss of connection.
```

Comment:
Ditto linebreaks.

Reply:
>If you want to update the Flatcar OS of particular nodes in a controlled fashion, there is a way to disable such updates. 

It seems to me this sentence is incorrect. Shouldn't it be "If you **don't** want to update"?

We should also mention that disabling updates is not recommended.

Reply:
Perhaps we could also document how to use hooks so nodes can be gracefully taken out from traffic etc.

Reply:
> If you want to update the Flatcar OS of particular nodes in a controlled fashion, there is a way to disable such updates.

This meant for manual updates. Why disabling updates is not recommended, until we provide a way to do it.

> Perhaps we could also document how to use hooks so nodes can be gracefully taken out from traffic etc.

Yes, but it is out of the scope of this PR. This PR concerns only with documenting the happy path updates. Once we have this foundation we can add more information on how to update it with nodes running 

- Networking apps (ingress, metallb)
- Storage apps (openebs, rook, self hosted storage)